src/apps/organisations/components/organisationCard.tsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import {
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  createStyles,
+  Grid,
+  makeStyles,
+  Theme,
+  Typography,
+} from "@material-ui/core";
+import {Link, useLocation} from "react-router-dom";
+
+interface Props {
+  name: string;
+  url: string;
+  index: number;
+}
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    containerName: {
+      overflowX: "auto",
+    },
+  })
+);
+const OrganisationCard: React.FC<Props> = ({
+  name,
+  url,
+  index
+}) => {
+  const classes = useStyles();
+  const location = useLocation();
+
+  return (
+    <Grid item xs={4}>
+      <Card>
+        <CardContent>
+          <Typography className={classes.containerName} noWrap variant='h5' data-testid={`name-${index}`}>
+            {name}
+          </Typography>
+          <Typography noWrap variant='body1' component='p' data-testid={`url-${index}`}>
+            {url}
+          </Typography>
```

Comment:
Personally, I don't think there's a great deal of value to adding this to the card. It would, however, be useful to display the `id` of the organisation.